### ISSUE TITLE
JOSE Review - comments on Introduction

### ISSUE THREAD

**Issue by kls2177** [atrisovic/weather-panel.github.io#76]
Nicely laid out. Just a few comments:
- The link to the IPCC report should be updated to AR6 (I realize that AR5 was the most up-to-date when this review started). You can link to the entire WGI landing page rather than a specific chapter. You may also consider adding a link to the WGII report (not just WGI) as this may be more relevant to your readers. Although, I agree that some WGI background knowledge is essential.
- For the region data section, do you want to add an example, e.g. census tract.
- Mathematical notation: as a climate scientist, I would prefer that T not be used as the dummy variable name as it may make the reader think of temperature. What about X? I haven't read the entire tutorial yet, so maybe the choice will make more sense later.

**Comment by jrising** [comment]
@kls2177 Thank you for the comments, across all of our sections! I have edited the Introduction under this commit:https://github.com/atrisovic/weather-panel.github.io/commit/2cc9796484c38addc214080081f0c20dc039e908

Responding to your comments individually:
 - Thanks for the note about AR6-- we definitely want to keep this up to date. I've linked to the AR6 WGI and WGII chapters, but I also included a comment pointing readers to WGI Chapter 1, since that provides the background information we think readers will particularly benefit from at this point in the tutorial.
 - I added examples for each region type: "political units (e.g. municipalities), statistical units (e.g., census tracts), and jurisdictions (e.g., educational districts)".
 - I switched this to X. It's somewhat common in the climate-economics field to use T as a proxy for any weather variable, since temperature is often the one that is of most interest, but I can see the potential for confusion. I clarified that we will use T and P for temperature and precip. elsewhere in the tutorial. We will also update the comments on functional forms to use X rather than T as our generic variable.

**Comment by kls2177** [comment]
@jrising 

Great updates!

**Comment by jrising** [comment]
@kls2177 Good point. I now changed all of the table of contents headings and within-section headings to use first-word-style capitalization.
